frontend/src/components/resultPage/SearchResultForm.js

```diff
@@ -872,6 +875,14 @@ export function SearchResults(props) {
     }
   };
 
+  const downloadFile = (fileName, base64Content, fileType) => {
```

Comment:
you can also use this logic [here](https://github.com/DIGI-UW/OpenELIS-Global-2/blob/develop/frontend/src/components/cytology/CytologyCaseView.js#L620-L626) to open the file in another window